homeassistant/components/open_meteo/weather.py

```diff
@@ -55,6 +60,15 @@ def __init__(
             name=entry.title,
         )
 
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Handle updated data from the coordinator."""
+        super()._handle_coordinator_update()
+        assert self.platform.config_entry
+        self.platform.config_entry.async_create_task(
+            self.hass, self.async_update_listeners(("daily",))
```

Comment:
Feels counter-productive if each integration has to implement this for things to work.

Reply:
Yeah it does, maybe we should do a CoordinatorWeatherEntity or something?

Reply:
Fixed, `OpenMeteoWeatherEntity` now extends `CoordinatorWeatherEntity`